indico/modules/events/settings.py

```diff
@@ -261,3 +262,9 @@ def get_themes_for(self, event_type):
     'emails': [],
     'phones': []
 })
+
+event_contribution_settings = EventSettingsProxy('contrib', {
```

Comment:
- `contribution_settings` for the proxy (when moving it)
- `'contributions'` as the first argument (the module name used when storing settings in the DB